src/components/task/TaskWrapper.js

```diff
@@ -0,0 +1,24 @@
+import {CodeEditor} from 'e-stella-code-editor'
+import {tasksApi} from "../../utils/apis/TasksApi";
+import {useParams} from "react-router-dom";
+
+export const TaskWrapper = ({ id: propId, toSolveTask = true, submitLeftOffset = 0 }) => {
+
+    let { id } = useParams()
+
+    if(!id) id = propId
```

Comment:
Co jeśli id będzie 0 (oczywiście tak nie będzie, ale still id jest int-em)? Ta logika wtedy nie działa

Reply:
1. Id może być czymkolwiek (to js), więc i tak wszystkiego nie obejdziemy.
2. To nie sysopy. Skoro id nie będzie 0, nie ma się czym martwić raczej.